bundles/org.openhab.binding.tado/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -209,20 +236,32 @@
 		<label>Swing</label>
 		<description>State of AC swing (only if supported by AC)</description>
 		<category>Flow</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Airflow</tag>
+		</tags>
 	</channel-type>
 
 	<channel-type id="light">
 		<item-type>Switch</item-type>
 		<label>Light</label>
 		<description>State of control panel light (only if supported by AC)</description>
 		<category>Light</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Light</tag>
+		</tags>
 	</channel-type>
 
 	<channel-type id="fanLevel">
 		<item-type>String</item-type>
 		<label>Fan Speed</label>
 		<description>AC fan level (only if supported by AC)</description>
 		<category>Fan</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>Speed</tag>
```

Comment:
`Speed` or `Level`? Description mentions level. I guess you can't control the speed in unit m/s, for example?

Reply:
We need to resolve this at the highest level. It in process of being discussed at length elsewhere. It relates to fans that are controlled off/low/medium/high. The tag could be speed, airflow, ventilation, level at least -- we have even seen one with RPM tagged as frequency. Personally I think speed is most appropriate. But honestly I would really appreciate if others can make proposals rather than asking questions.

Reply:
Given my other comment below about `Switch.Power` implying switch on or off, I think therefore `Control.Speed` should be the tag for controlling a "speed-not-necessarily-in-metres-per-second" object.